src/toil_scripts/gatk_germline/germline.py

```diff
@@ -42,10 +42,11 @@
 
 import yaml
 from toil.job import Job
+from toil_lib import require
```

Comment:
I ran optimize imports before realizing it messed up the `align_and-call` pipeline's imports, so I just did a quick find-and-replace. I'll go back and do that.